src/pages/simulator/index.tsx

```diff
@@ -0,0 +1,119 @@
+import { Link, useParams, useSearch } from '@tanstack/react-router';
+import { buttonStyles } from 'components/common/button/buttonStyles';
+import { SimInputChart } from 'components/simulator/input/SimInputChart';
+import { SimInputOverlapping } from 'components/simulator/input/SimInputOverlapping';
+import { SimInputRecurring } from 'components/simulator/input/SimInputRecurring';
+import { SimInputStrategyType } from 'components/simulator/input/SimInputStrategyType';
+import { SimInputTokenSelection } from 'components/simulator/input/SimInputTokenSelection';
+import dayjs from 'dayjs';
+import { useSimulatorInput } from 'hooks/useSimulatorInput';
+import { useEffect, useRef, useState } from 'react';
+import { useModal } from 'hooks/useModal';
+import { useStore } from 'store';
+import { cn } from 'utils/helpers';
+
+export const SimulatorPage = () => {
+  const { simDisclaimerLastSeen, setSimDisclaimerLastSeen } = useStore();
+  const [timeRange] = useState({
+    start: dayjs().unix() - 60 * 60 * 24 * 30 * 12,
+    end: dayjs().unix(),
+  });
+
+  const { simulationType } = useParams({ from: '/simulator/$simulationType' });
+  const searchState = useSearch({
+    from: '/simulator/$simulationType',
+  });
+  const { dispatch, state, bounds } = useSimulatorInput({ searchState });
+
+  const [initBuyRange, setInitBuyRange] = useState(true);
+  const [initSellRange, setInitSellRange] = useState(true);
+
+  const { openModal } = useModal();
+  const hasOpenedDisclaimer = useRef(false);
+
+  const inputError =
+    Number(state.buy.budget) + Number(state.sell.budget) <= 0
+      ? 'Please add Sell and/or Buy budgets'
+      : null;
+
+  useEffect(() => {
+    if (
+      !!simDisclaimerLastSeen &&
+      simDisclaimerLastSeen > dayjs().unix() - 15 * 60 * 1000
```

Comment:
I believe you are making a mistake - daysjs unix is in seconds. I assume you want 15 minutes. not 15,000 :)
Please do this:
`dayjs().subtract(15, 'minute').unix()`